cpp/CMakeLists.txt

```diff
@@ -58,6 +58,9 @@ include(PythonUtils) # Must be called before Pybind (third_party) to override it
 
 add_subdirectory(third_party)
 add_subdirectory(proto)
+if(NOT ${ARCTICDB_USING_CONDA})
+    include(AzureVcpkg) #AzureVcpkg.cmake is from https://github.com/Azure/azure-sdk-for-cpp/blob/main/cmake-modules/AzureVcpkg.cmake, commit ada77b3
```

Comment:
It would be better to put the comment in `AzureVcpkg.cmake` itself.

Reply:
I think it is better to leave it is as the file should be kept identical as the one in Azure SDK. As later this file may need to be updated, along with the SDK.